keyboards/tr60w/keymaps/default/keymap.c

```diff
@@ -0,0 +1,38 @@
+#include "tr60w.h"
```

Comment:
This should be changed to:

```suggestion
#include QMK_KEYBOARD_H
```